lib/Controller/QueueController.php

```diff
@@ -83,8 +83,6 @@ public function __construct(
 
 
 	/**
-	 * // TODO: Delete the NoCSRF check
-	 *
 	 * @PublicPage
 	 * @NoCSRFRequired
 	 * @NoAdminRequired
```

Comment:
PublicPage makes No(Sub)AdminRequired obsolete

Reply:
As said, I will make some more test after every PRs are merged regarding the CSRF.
Here, I need to check if doing local curl request will work without the @NoCSRFRequired.

I will also clean the No(Sub)AdminRequired in another PR

Reply:
From where is this method requested?

Reply:
The QueueController is used to async process. The request is done from the app itself, locally.